client/scripts/DrawTogether.js

```diff
@@ -1609,18 +1718,37 @@ DrawTogether.prototype.createFavorite = function (x, y, name) {
 };
 
 DrawTogether.prototype.createProtectedRegion = function (from, to) {
-	if (!this.memberlevel) {
-		this.chat.addMessage("Creating a protected region is a member only feature.");
+	if (typeof this.reputation === 'undefined') { 
+		this.chat.addMessage("You must be logged in to create protected regions.");
 		return;
 	}
+	var regionCount = this.regionsList.length || 0;
+
+	if (!this.memberlevel) {
+		if(this.reputation < this.REGION_MIN_REP){
+			
+			this.chat.addMessage("You must have at least "+ this.REGION_MIN_REP +" rep!");
+			return;
+		}
+		if(typeof this.regionsList !== 'undefined')
+		{
+			if(typeof this.regionsList.length === 'number')
```

Comment:
If whatever you are doing the if for is more than one line then I don't like this style. You should include the brackets.